src/eos-updater-ctl

```diff
@@ -388,6 +389,15 @@ def main():
     parser_update = subparsers.add_parser('update',
                                           help='run the entire update process')
     parser_update.set_defaults(function=command_update)
+    parser_update.add_argument('--force', action='store_true',
+                               help='force fetching even if on a metered '
+                                    'network connection (this only affects '
+                                    'the fetch stage)')
+    parser_update.add_argument('--timeout', type=nonnegative_int, default=0,
```

Comment:
Nah, since it’s a wholesale lift of all the options from `fetch` to `update`. It should be mentioned in the commit message though. I’ll fix that and push.